dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/relationship/RelationshipDeletionHandlerTest.java

```diff
@@ -87,15 +87,16 @@ void allowDeleteRelationshipTypeWithoutData() {
 
   @Test
   void deleteTrackedEntityInstance() {
-    when(relationshipService.getRelationshipsByTrackedEntityInstance(any(), anyBoolean()))
+    when(relationshipService.getRelationshipsByTrackedEntityInstance(
+            any(), anyBoolean(), anyBoolean()))
         .thenReturn(singletonList(new Relationship()));
 
     ObjectDeletionRequestedEvent event =
         new ObjectDeletionRequestedEvent(new TrackedEntityInstance());
     deletionManager.onDeletion(event);
 
     verify(relationshipService, atLeastOnce())
-        .getRelationshipsByTrackedEntityInstance(any(), anyBoolean());
+        .getRelationshipsByTrackedEntityInstance(any(), anyBoolean(), anyBoolean());
```

Comment:
`anyBoolean` can be replaced with false (or whatever representation of false is ok for mockito here), as the value will always be false in the deletion handler right?